go.mod

```diff
@@ -3,15 +3,15 @@ module github.com/ElrondNetwork/elrond-go
 go 1.13
 
 require (
-	github.com/ElrondNetwork/arwen-wasm-vm/v1_2 v0.0.0-20210729074338-530860a14ea8
-	github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v0.0.0-20210729081242-76a2ad8addc3
-	github.com/ElrondNetwork/arwen-wasm-vm/v1_4 v0.0.0-20210728145300-26a2a7e48775
+	github.com/ElrondNetwork/arwen-wasm-vm/v1_2 v1.2.28
+	github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.27
+	github.com/ElrondNetwork/arwen-wasm-vm/v1_4 v1.4.3
```

Comment:
this is not latest, that is why it works without defining managed buffers. But this won't reference the last changes in vmcommon.